packages/flutter/test/widgets/text_golden_test.dart

```diff
@@ -736,13 +736,13 @@ void main() {
                           child: Text.rich(
                             TextSpan(
                               text: 'inner',
-                              style: TextStyle(color: Color(0xff402f4ff)),
+                              style: TextStyle(color: Color(0xf402f4ff)),
```

Comment:
Does this pass?

I suspect the intention here was to have full alpha and drop that last f, but hard to say.

Reply:
Yes, it passes locally. I didn't really know what was expected but as `0xff402f4ff` is treated as 32 bits integer it's logically truncated on left.